Pod/Classes/PINRemoteImageManagerResult.h

```diff
@@ -7,8 +7,13 @@
 //
 
 @import UIKit;
-
+#if __has_include(<FLAnimatedImage/FLAnimatedImage.h>)
 #import <FLAnimatedImage/FLAnimatedImage.h>
+#define USE_FLANIMATED_IMAGE    1
+#else
+#define USE_FLANIMATED_IMAGE    0
+#define FLAnimatedImage         NSObject
```

Comment:
I'm not sure I like this idea of this alias, especially because this `#define` will leak outside of the PINRemoteImage library to anything that includes its headers.